Source/UI/Coordinators/PasswordCoordinator.swift

```diff
@@ -308,27 +308,35 @@ extension PasswordCoordinator {
             completion()
         } else {
             Settings.setValue(true, forKey: hasLoggedInBeforeSettingsKey)
-            let viewModel = PasswordViewModel(
-                identifier: identifier,
-                loginFlowVariant: loginFlowVariant,
-                localizationBundle: self.configuration.localizationBundle
-            )
-            let message = viewModel.biometricsOnboardingMessage
-                .replacingOccurrences(of: "$0", with: configuration.appName)
-            let title = viewModel.biometricsOnboardingTitle
+            let context = LAContext()
+            context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil)
+            if context.biometryType == .touchID {
+                let viewModel = PasswordViewModel(
+                    identifier: identifier,
+                    loginFlowVariant: loginFlowVariant,
+                    localizationBundle: self.configuration.localizationBundle
+                )
+                let message = viewModel.biometricsOnboardingMessage
+                    .replacingOccurrences(of: "$0", with: configuration.appName)
+                let title = viewModel.biometricsOnboardingTitle
```

Comment:
Suggestion: Since the text in the title and message is dedicated to `TouchID`, you can use names: `touchIdOnboardingTitle` and `touchIdOnboardingMessage`